ConsoleApp1/ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace TeleprompterConsole
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            RunTeleprompter().Wait();
+        }
+        private static async Task ShowTeleprompter(TelePrompterConfig config)
```

Comment:
Space after a method pls

Should read

```cs
static void Main()
{
    //stuff
}

static void NextMethod()
{ 
    //stuff
}
```

Reply:
Yep - I do know that.  An oversight in this case.